tilequeue/stats.py

```diff
@@ -0,0 +1,63 @@
+class TileProcessingStatsHandler(object):
+
+    def __init__(self, stats):
+        self.stats = stats
+
+    def __call__(self, coord_proc_data):
+        with self.stats.pipeline() as pipe:
+            pipe.timing('process.fetch',
+                        coord_proc_data.timing['fetch_seconds'])
+            pipe.timing('process.process',
+                        coord_proc_data.timing['process_seconds'])
+            pipe.timing('process.upload', coord_proc_data.timing['s3_seconds'])
+            pipe.timing('process.ack', coord_proc_data.timing['ack_seconds'])
+            pipe.timing('process.time-in-queue',
+                        coord_proc_data.timing['queue'])
+
+            for layer_name, features_size in coord_proc_data.size.items():
+                metric_name = 'process.size.%s' % layer_name
+                pipe.gauge(metric_name, features_size)
+
+            pipe.incr('process.storage.stored',
+                      coord_proc_data.store_info['stored'])
+            pipe.incr('process.storage.skipped',
+                      coord_proc_data.store_info['not_stored'])
+
+
+class RawrTileEnqueueStatsHandler(object):
+
+    def __init__(self, stats):
+        self.stats = stats
+
+    def __call__(self, n_coords, n_payloads, n_msgs_sent):
+        with self.stats.pipeline() as pipe:
+            pipe.gauge('rawr.enqueue.coords', n_coords)
+            pipe.gauge('rawr.enqueue.groups', n_payloads)
+            pipe.gauge('rawr.enqueue.calls', n_msgs_sent)
```

Comment:
This name is a little opaque. It represents the number of function calls we make to the underlying sqs queue 
 client. Each call will have up to 10 payloads which correspond to sqs messages, and each payload can have multiple tiles in it, which are z10 groupings.

Reply:
This is the number of SQS API calls?

Reply:
Yea, exactly.